client/web/src/search/results/DidYouMean.tsx

```diff
@@ -188,13 +188,7 @@ export const ServerQueryAssist: React.FunctionComponent<React.PropsWithChildren<
             <H3>
                 Also showing results for:
                 <Tooltip content="We returned all the results for your query. We also added results you might be interested in for similar queries. Below are similar queries we ran.">
-                    <Icon
-                        size="sm"
-                        className="ml-1"
-                        as={InformationOutlineIcon}
-                        tabIndex={0}
-                        aria-label="Showing results for alternative queries"
-                    />
+                    <Icon size="sm" className="ml-1" as={InformationOutlineIcon} tabIndex={0} aria-hidden={true} />
```

Comment:
Rethinking this, if it's focusable, I think it should not be aria-hidden, but rather labelled to convey the same message as the icon:

```suggestion
                    <Icon size="sm" className="ml-1" as={InformationOutlineIcon} tabIndex={0} aria-label="More information" />
```

Reply:
sounds good!